python/taichi/lang/ops.py

```diff
@@ -185,36 +185,92 @@ def _ternary_operation(taichi_op, python_op, a, b, c):
 
 @unary
 def neg(a):
+    """Negate function.
+
+    Args:
+        a: a.
+
+    Returns:
+        -a.
+    """
```

Comment:
```suggestion
    """Negate function.

    Args:
        a: A number.

    Returns:
        The negative value of number a.
    """
```
```suggestion
        
```
Explaining with some words?